scripts/get-test-pass-rate.js

```diff
@@ -0,0 +1,50 @@
+// This script is used in the "runAfterAll" stage in our E2E test pipeline. It's used 
+// to get timeline and metrics data so that the test pass rate can be calculated in a later step.
+
+const BUILD_ID = process.env.BUILD_ID;
+const ADO_API_TOKEN = process.env.ADO_API_TOKEN;
+const TEST_WORKSPACE = process.env.TEST_WORKSPACE;
+const BUILD_SOURCES_DIRECTORY = process.env.BUILD_SOURCES_DIRECTORY;
+
+// Create output folder - Note: This requires Node.js fs module
+const fs = require('fs');
+console.log("Starting fetch")
```

Comment:
Nit
```suggestion
console.log("Starting fetch");
```

Reply:
We must not have prettier running against this directory. It might be a good idea to run `npx prettier ./scripts/get-test-pass-rate.js` from the repo root before merging this to ensure our style guidelines are followed.

Reply:
Did not know you could run prettier like this. Thanks for the tip